gp-includes/things/translation.php

```diff
@@ -661,6 +661,23 @@ public function last_modified( $translation_set ) {
 		return $last_modified;
 	}
 
+	/**
+	 * Get IDs of translations by ID of their original and translation set.
+	 *
+	 * @since 2.3.0
+	 *
+	 * @param int $original_id        ID of original.
+	 * @param int $translation_set_id ID of translation set.
+	 * @return array $ids An array of IDs of matching translations.
+	 */
+	public function ids_by_original_and_set( $original_id, $translation_set_id  ) {
+		global $wpdb;
+
```

Comment:
Yes and no.  Yes to "always display link" and no/maybe to "default" ;)

On small to medium sites the history link is probably not used by most translators, so the overhead of the extra SQL probably outweighs the benefit.

Of course on small/medium sites the overhead might not be significant enough to be an issue.

Perhaps @ocean90 could do some stats on the offline version of translate.w.org he has and see how much of an impact it really is?